src/query/ast/src/parser/stage.rs

```diff
@@ -213,5 +213,9 @@ pub fn select_stage_option(i: Input) -> IResult<SelectStageOption> {
             rule! { PATTERN ~ "=>" ~ #literal_string },
             |(_, _, pattern)| SelectStageOption::Pattern(pattern),
         ),
+        map(
+            rule! { FILE_FORMAT ~ "=>" ~ #literal_string },
```

Comment:
It would be better to use enum instead of string? For example:
```
select * from @stage1  (FILE_FORMAT => PARQUET)
```

Reply:
we will use this syntax in infer_schema too.
in general (and in snow) the value of `FILE_FORMAT` should be a format_name created by `create format`.
after discussion with @sundy-li, we choose to allow build-in format_name too and forbid use to create format parquet.

infer_schema is a table function,  in which a value of named params (after `=> `) need to be a expr.
while `PARQUET`(without quote) is an ident, can not be parsed here.